### PR TITLE
fix liquidity present value tests

### PR DESCRIPTION
**Pull request by dpaiton**
initial present value is recalculated for add & remove liquidity

### CODE REVIEW

File: lib/agent0/agent0/interactive_fuzz/fuzz_present_value.py [hunk 1]
```diff
@@ -118,14 +121,37 @@ def fuzz_present_value(
...
+                )
                 trade_amount = FixedPoint(
                     scaled_value=int(
                         np.floor(rng.uniform(low=min_trade.scaled_value, high=agent.wallet.balance.amount.scaled_value))
                     )
                 )
                 trade_event = agent.add_liquidity(trade_amount)
             case HyperdriveActionType.REMOVE_LIQUIDITY:
+                # recompute initial present value for liquidity actions
+                check_data["initial_present_value"] = interactive_hyperdrive.hyperdrive_interface.calc_present_value(
```
Comment (slundqui) [inline lib/agent0/agent0/interactive_fuzz/fuzz_present_value.py:152]: Can you double check this? I think this value is getting overwritten: this loop first executes add liquidity and sets this parameter, then removes liquidity and resets this parameter.

Comment (dpaiton) [reply lib/agent0/agent0/interactive_fuzz/fuzz_present_value.py:152]: the loop recalculates the initial before the trade for each trade; so the steps are
- calculate initial present value
- add liquidity
- check new pv against initial

- calculate initial pv
- remove liquidity
- check new pv against initial

Comment (dpaiton) [reply lib/agent0/agent0/interactive_fuzz/fuzz_present_value.py:152]: I could remove the loops entirely, they are kind of overcomplicating parsing the test

Comment (slundqui) [reply lib/agent0/agent0/interactive_fuzz/fuzz_present_value.py:152]: Ah okay, I see, didn't realize we check invariances at the end of every loop. In this case, this var should be set outside of the match

File: lib/agent0/agent0/interactive_fuzz/fuzz_present_value.py [hunk 2]
```diff
@@ -118,14 +121,37 @@ def fuzz_present_value(
             case HyperdriveActionType.CLOSE_SHORT:
                 maturity_time, open_trade = next(iter(agent.wallet.shorts.items()))
                 trade_event = agent.close_short(maturity_time=maturity_time, bonds=open_trade.balance)
+            case _:
+                raise ValueError(f"Invalid {trade_type=}")
```
Comment (slundqui) [inline lib/agent0/agent0/interactive_fuzz/fuzz_present_value.py:125]: I believe these trades are not getting checked in invariances, this for loop is simply opening trades. A quick scan shows there are indeed checks against these trades

Comment (dpaiton) [reply lib/agent0/agent0/interactive_fuzz/fuzz_present_value.py:125]: yeah I just caught that too. one sec

**Review by slundqui — APPROVED** [review]
LGTM

**Pull request merged**